test/yargs.js

```diff
@@ -886,6 +887,43 @@ describe('yargs dsl tests', function () {
       argv.dotNotation.should.equal(false)
     })
   })
+
+  describe('skipValidation', function () {
+    it('skips validation if an option with skipValidation is present', function () {
+      var argv = yargs(['--koala', '--skip'])
+          .demand(1)
+          .fail(function (msg) {
+            expect.fail()
+          })
+          .skipValidation(['skip', 'reallySkip'])
+          .argv
+      argv.koala.should.equal(true)
+    })
+
+    it('does not skip validation if no option with skipValidation is present', function (done) {
+      var argv = yargs(['--koala'])
+          .demand(1)
+          .fail(function (msg) {
+            return done()
+          })
+          .skipValidation(['skip', 'reallySkip'])
+          .argv
+      argv.koala.should.equal(true)
+    })
+
+    it('allows key to be specified with option shorthand', function () {
```

Comment:
it would be worthwhile to add one additional test that shows validation failing if one skipped argument is provided, but another argument is provided that causes it to still become a failure scenario.